src/sdl/window.cpp

```diff
@@ -54,6 +54,12 @@ window::window(const std::string& title,
 	// Set default blend mode to blend.
 	SDL_SetRenderDrawBlendMode(*this, SDL_BLENDMODE_BLEND);
 
+	// Use linear scaling when rendering, if applicable.
+	SDL_SetHint(SDL_HINT_RENDER_SCALE_QUALITY, "linear");
+
+	// Enable 3D rendering for the window surface. Not sure if this really does anything.
+	SDL_SetHint(SDL_HINT_FRAMEBUFFER_ACCELERATION, "1");
```

Comment:
This is setting the defaults, you're better off omitting it as you may break things by trying to override it on an unsupported platform.